plugins/modules/swim_intent.py

```diff
@@ -1099,12 +1116,13 @@ def get_diff_distribution(self):
 
         if len(device_uuid_list) == 0:
             self.status = "failed"
-            msg = "No devices found for Image Distribution"
-            self.result['msg'] = msg
-            self.log(msg)
+            self.msg = "No devices found for Image Distribution"
```

Comment:
`    self.message = "Image Distribution cannot proceed due to the absence of device(s)"
`

Reply:
added.